lib/ansible/modules/crypto/luks_device.py

```diff
@@ -541,6 +563,7 @@ def run_module():
         keysize=dict(type='int'),
         label=dict(type='str'),
         uuid=dict(type='str'),
+        type=dict(type='str', choices=['luks1', 'luks2']),
     )
```

Comment:
Add some code below `module = AnsibleModule(...` which checks for `module.params['label'] is not None and module.params['type'] == 'luks1'`, and if that happens, calls `module.fail_json(msg='You cannot combine type luks1 with the label option.')`.

Reply:
done